### PR TITLE
reranker id added in config

### PR DESCRIPTION
**Pull request by adeelehsan**

### CODE REVIEW

File: src/contexts/ConfigurationContext.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ import {
   UxMode,
   normal_reranker_id,
   mmr_reranker_id,
-  FcsMode, FCS_MODE,
+  FcsMode, FCS_MODE, slingshot_reranker_id
```
Comment (ofermend) [inline src/contexts/ConfigurationContext.tsx:16]: Why do you need the slingshot_reranker_id as a config param? I thought it could be just in the code, no?

Comment (adeelehsan) [reply src/contexts/ConfigurationContext.tsx:16]: its in code, just importing in the config to set the reranker id for https://github.com/vectara/vectara-answer/pull/155/files#diff-3a445224bc341427785ac65b7124070512bb1920fdd07cb979a4deb9d7f87bf4R157

File: src/contexts/ConfigurationContext.tsx [hunk 2]
```diff
@@ -476,6 +475,23 @@ export const ConfigContextProvider = ({ children }: Props) => {
...
     }
 
+    const isRankerEnabled = (rerankerName: string | undefined) => {
+      return rerankerName === "default" || rerankerName === "slingshot"
+        || rerankerName === "mmr" || false
+    }
+    const getRerankerId = (rerankerName: string | undefined) => {
+      if (rerankerName === "mmr")  return mmr_reranker_id
+      else if (rerankerName === "slingshot") return slingshot_reranker_id
+      else return normal_reranker_id
```
Comment (ofermend) [inline src/contexts/ConfigurationContext.tsx:485]: I think the name "default" does not make sense to me in the reranker name. We should figure out it's name.
Default seems to indicate it's used by default, and that is not true in this case. If unspecified then it's going to use what is called here "normal" reranker.

Comment (adeelehsan) [reply src/contexts/ConfigurationContext.tsx:485]: Yeah that makes sense.

File: README.md [hunk 3]
```diff
@@ -265,20 +265,16 @@ This means the user will only be able to select a specific source for each query
 Whether to use Vectara's [reranking](https://docs.vectara.com/docs/api-reference/search-apis/reranking) functionality. Note that reranking currently works for English language only, so if the documents in your corpus are in other languages, it's recommended to set this to "False".
 
 ```yaml
-# Reranking enabled: True or False
-rerank: False
+# choose a reranker from the following choices
+rerank_name: default | slingshot | mmr
```
Comment (ofermend) [inline README.md:269]: Should this say "reranker_name" not "rerank_name"?

**Pull request merged**